.github/workflows/release.yml

```diff
@@ -0,0 +1,130 @@
+name: Release
+
+on:
+  push:
+    tags:
+      # For root tags, such as v0.4.2
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+      # For subfolder tags, such as workflow-engine-v1.18.0
+      #- "[a-zA-Z-_]+v[0-9]+.[0-9]+.[0-9]+"
+      #- "[a-zA-Z-_]+v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+
+jobs:
+  build:
+    permissions:
+      id-token: write
+      contents: read
+      attestations: write
+    strategy:
+      matrix:
+        include:
+          - platform: linux/amd64
+            runner: ubuntu-latest
+            asset_name: ${{ github.event.repository.name }}-linux-amd64-latest
+          - platform: linux/arm64
+            runner: arm-ubuntu-latest-8core
```

Comment:
Is this one of the expensive runners or are there no arm runners with unspecified numbers of cores?